forward_engineering/services/statements/getPrimaryKeySampleByStructure.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @typedef {import('../../../shared/types').PkSegment} PkSegment
+ * @typedef {import('../../../shared/types').UUID} UUID
+ */
+const RandExp = require('randexp');
+const { PK_SEGMENT_TYPE } = require('../../../shared/constants');
+
+/**
+ * @param {{ collection: object, jsonData: object }}
+ * @returns {string}
+ */
+const getPrimaryKeySampleByStructure = ({ collection, jsonData }) => {
+	const keyField = Object.values(collection.properties || {}).find(field => field.primaryKeyStructure);
+	const primaryKeyStructure = keyField?.primaryKeyStructure;
+
+	if (!Array.isArray(primaryKeyStructure)) {
+		return '';
+	}
+
+	return primaryKeyStructure.reduce((result, segment) => {
+		switch (segment.segmentType) {
+			case PK_SEGMENT_TYPE.field: {
+				const segmentValue = getPrimaryKeyStructureFieldValue({ segment, collection, jsonData });
+				return result + segmentValue;
+			}
+			case PK_SEGMENT_TYPE.pattern: {
+				const segmentValue = getPrimaryKeyStructurePatternValue({ segment });
+				return result + segmentValue;
+			}
+			case PK_SEGMENT_TYPE.constant:
+			case PK_SEGMENT_TYPE.separator: {
+				return result + (segment.segmentValue ?? '');
+			}
+			default:
+				return result;
+		}
+	}, '');
+};
+
+/**
+ * @param {{ segment: PkSegment, collection: object, jsonData: object }} param0
```

Comment:
Please, stick to one approach, use "param" as a method parameters name in JSDoc or do not use the name at all as you did in method above